src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/UnitPage.tsx

```diff
@@ -0,0 +1,86 @@
+import type {RouteProp} from '@react-navigation/native';
+import React, {useMemo} from 'react';
+import SelectionList from '@components/SelectionList';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyOnyxProps} from '@pages/workspace/withPolicy';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type OptionRow = {
+    value: string;
+    text: string;
+    keyForList: string;
+    isSelected: boolean;
+};
+
+type WorkspaceUnitPageProps = WithPolicyOnyxProps & {
+    route: RouteProp<{params: {policyID: string; unit?: string; rate?: string}}>;
+};
+
+function WorkspaceUnitPage(props: WorkspaceUnitPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const unitItems = useMemo(
+        () => ({
+            [CONST.CUSTOM_UNITS.DISTANCE_UNIT_KILOMETERS]: translate('common.kilometers'),
+            [CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES]: translate('common.miles'),
+        }),
```

Comment:
The first letter of each item on this page should be capitalised to maintain consistency with other list views.